NewsApp/app/src/main/java/com/thomasvondollen/newsapp/MainActivity.kt

```diff
@@ -66,45 +18,72 @@ class MainActivity : AppCompatActivity() {
         val view = binding.root
         setContentView(view)
 
-        //looping through Article objects and assigning contents to TextViews
-        var count = 0
-
-        for(item in binding.mainGroup.children) {
+        val newsService = NewsService()
+        val newsArticles = newsService.createArticleArray()
 
-            var contents = ""
 
-            val currentArticle = dummyArticles[count]
+        Log.i("MyTag",newsArticles[0].title)
+        Log.i("MyTag",newsArticles[2].author)
```

Comment:
Nit: but if you won't be needing these values posted to the console, you could remove these two log statements.